### ISSUE TITLE
Collections configurable facets accessibility concerns

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#551]
### Descriptive summary

#549 Added orderable facet configuration for collections. The drag-and-drop design for ordering isn't very keyboard accessible and is problematic for screen readers.
https://salesforce-ux.github.io/dnd-a11y-patterns/#/sortA?_k=z62gvk provides an example that should work well for screen readers and improves the keybindings.

### Expected behavior

Improved keybindings and audio feedback for screen readers.

### Related work

#549, #528, https://salesforce-ux.github.io/dnd-a11y-patterns/#/sortA?_k=z62gvk

### Accessibility Concerns

Screen reader feedback
Keyboard only use

**Comment by CGillen** [comment]
#549 included a fix for this

QA:
- [ ] Can "pick-up" facet with focus+spacebar/enter
- [ ] Can move "picked-up" facet with up/left or down/right
- [ ] Can "drop" facet with spacebar/enter
- [ ] Can have multiple facets selected
- [ ] Selected facet is distinguishable
- [ ] Can only move selected facet

**Comment by raybrarian** [comment]
Hey @CGillen, I'm QAing this trying shift+space, ctrl+m, ctrl+tab to select multiples with no luck - clearly I don't know what the right shortcut is! If you could drop that in here, that would be great! One FYI - using ctrl+m means that the focus style change to the element persists, but the focus doesn't: 
<img width="418" alt="focus-persistence" src="https://user-images.githubusercontent.com/4460510/59461339-051fd200-8dd6-11e9-9547-16c97f08e47a.png">

**Comment by CGillen** [comment]
Ah, a bit of an ambiguous direction. When I said multiple selected, I meant multiple with the "focus style." Basically you can move more than one facet at any given moment, but not at the same time.

So, you should be able to tab to a facet, pick it up with space, tab to another, pick up that one too, and start moving the second one, tab back to the first facet, and move it too. Those two will not move at the same time though. Either should be able to be put down independently of the other.

**Comment by raybrarian** [comment]
Ah, gotcha - so the focus stays until you essentially go back and release it, whether or not you move it anywhere. But no matter which facets have received focus, it's only the current one which will move. I've updated the QA checklist above to capture that!

QA pass.
